DEPLOYMENT.md

```diff
@@ -0,0 +1,56 @@
+## Initial Setup:
+
+These were the steps I followed when setting up the web server for the help desk. If for any reason the help desk needs to be redeployed follow these instructions.
+
+1. Launch Ubuntu 20.04 LTS instance from AWS console
```

Comment:
These steps look good — could you include some screenshots, or at least for the first step?